src/repository/cert.rs

```diff
@@ -1480,7 +1480,7 @@ impl TbsCert {
     /// the value of the DER-encoded bit string of the subject public key.
     ///
     /// Conforming CAs MUST mark this extension as non-critical.
-    fn take_subject_key_identifier<S: decode::Source>(
+    pub fn take_subject_key_identifier<S: decode::Source>(
```

Comment:
Do you need this `pub` or would something like `pub(crate)` be enough?

Reply:
`pub(crate)` is enough. Fixed.